pkg/kubelet/apis/cri/v1alpha1/runtime/api.pb.go

```diff
@@ -400,7 +400,7 @@ type Mount struct {
 	Readonly bool `protobuf:"varint,3,opt,name=readonly,proto3" json:"readonly,omitempty"`
 	// If set, the mount needs SELinux relabeling.
 	SelinuxRelabel bool `protobuf:"varint,4,opt,name=selinux_relabel,json=selinuxRelabel,proto3" json:"selinux_relabel,omitempty"`
-	// Requested propagation mode.
+	// Requested propagation mode. Optional.
```

Comment:
Is the CRI defining a default for this?

Reply:
MountPropagation is an enum, so it will be default to zero, which means no mount propagation ("private" in Linux terminology).

And for other optional fields, their default values are following golang's default value with the types.

Reply:
proto3 doesn't allow custom default values.